packages/deployment-service/src/httpApi.ts

```diff
@@ -0,0 +1,60 @@
+import { NestFactory } from '@nestjs/core'
+import { INestApplication, ValidationPipe } from '@nestjs/common'
+import { AppModule } from './app-module'
+import { ExpressAdapter } from '@nestjs/platform-express'
+import { createServer, proxy } from 'aws-serverless-express'
+import { Handler, Context, APIGatewayEvent } from 'aws-lambda'
+import { eventContext } from 'aws-serverless-express/middleware'
+import { Server } from 'http'
+import express, { Express } from 'express'
+import * as bodyParser from 'body-parser'
+import { CorsHeaderInterceptor } from '@reapit/utils-nest'
+import { APIGatewayProxyResult } from 'aws-lambda/trigger/api-gateway-proxy'
+const crypto = require('crypto')
+
+global.crypto = crypto
```

Comment:
needed?

Reply:
sort of? Will need to refactor the auth method tbh